internal/internal_task_pollers.go

```diff
@@ -910,7 +910,7 @@ func (atp *activityTaskPoller) ProcessTask(task interface{}) error {
 		return reportErr
 	}
 
-	activityMetricsScope.Timer(metrics.ActivityEndToEndLatency).Record(time.Since(activityTask.pollStartTime))
```

Comment:
Also, this does not seem to be the latest change. Not sure why it's shown this way in the PR.